src/pages/signInV2.vue

```diff
@@ -251,6 +260,52 @@ async function handleSubmit() {
   handlePasswordlessLoginRequest(userEmailInput.value)
 }
 
+async function handleBearerLoginRequest(
+  type: BearerAuthentication,
+  _data: unknown
+) {
+  // Intentionally doing this to get the KeyReconstructor out of Auth SDK (which we shouldn't expose to external users)
+  // but would be OK to use internally
+  const ap = await getAuthProvider(app.id)
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  await ap.initKeyReconstructor()
+  const kr = // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    ap.keyReconstructor
+
+  switch (type) {
+    case BearerAuthentication.firebase: {
+      const data = _data as {
+        uid: string
+        token: string
+      }
+
+      const info = await kr.getPrivateKey({
+        verifier: BearerAuthentication.firebase,
+        id: data.uid,
+        idToken: data.token,
+      })
+
+      // TODO
```

Comment:
What is left here? @mmjee

Reply:
Error handling, treating Firebase differently. This is probably the hackiest feature we have.